pkg/i2gw/providers/gce/ir_converter_test.go

```diff
@@ -36,25 +38,33 @@ import (
 	gatewayv1 "sigs.k8s.io/gateway-api/apis/v1"
 )
 
-func Test_convertToIR(t *testing.T) {
-	testNamespace := "default"
-	testHost := "test.mydomain.com"
-	testServiceName := "test-service"
-	testBackendConfigName := "test-backendconfig"
-	iPrefix := networkingv1.PathTypePrefix
-	implSpecificPathType := networkingv1.PathTypeImplementationSpecific
+const (
+	testNamespace         = "default"
+	testHost              = "test.mydomain.com"
+	testServiceName       = "test-service"
+	testBackendConfigName = "test-backendconfig"
 
-	gPathPrefix := gatewayv1.PathMatchPathPrefix
-	gExact := gatewayv1.PathMatchExact
+	saTypeClientIP = "CLIENT_IP"
+	saTypeCookie   = "GENERATED_COOKIE"
+)
 
+var (
+	// These variables are referenced by pointers.
```

Comment:
Can simplify a lot of this with a func like https://github.com/kubernetes-sigs/gateway-api/blob/b5e115d06513555c7bc81bf7535f896105aac450/pkg/test/cel/main_test.go#L60-L62.

Reply:
Updated, thanks!